World/Updates/Rel21/Rel_21_14_087_Multiple_NPC_Fixes.sql

```diff
@@ -0,0 +1,143 @@
+        -- UPDATE THE DB VERSION
+        -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+        INSERT INTO `db_version` VALUES (@cNewVersion, @cNewStructure, @cNewContent, @cNewDescription, @cNewComment);
+        SET @cNewResult := (SELECT description FROM db_version WHERE `version`=@cNewVersion AND `structure`=@cNewStructure AND `content`=@cNewContent);
+
+        -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+        -- -- PLACE UPDATE SQL BELOW -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
+        -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+			-- Farmer Solliden update run speed and rank
+				UPDATE `creature_template` SET `Rank`='2', `SpeedRun`='1.14286' WHERE  `Entry`=1936;
+
+			-- Surf Crawler has wrong inhabit type resulting in constant Evade Bug
+				UPDATE `creature_template` SET `InhabitType`='3' WHERE  `Entry`=3107;
+
+			-- School of Fish will sometimes fly due to wrong InhabitType
+				UPDATE `creature_template` SET `InhabitType`='2' WHERE  `Entry`=3107;
+
+			-- Morin Cloudstalker move between waypoints
+				UPDATE `creature_template` SET `MovementType`='2' WHERE  `Entry`=2988;
+				UPDATE `creature` SET `MovementType`=2, `modelid`=1624 WHERE `guid`=26577;
+				DELETE FROM `creature_movement_template` WHERE `entry`=2988;
+				INSERT INTO `creature_movement_template` (entry, point, position_x, position_y, position_z, orientation, waittime, script_id) VALUES
+					(2988,1,-2368.63,-890.525,-9.42405,100,0,0),
+					(2988,2,-2360.93,-832.411,-9.34033,100,0,0),
+					(2988,3,-2354.42,-804.029,-9.42432,100,0,0),
+					(2988,4,-2344.20,-749.717,-9.42432,100,0,0),
+					(2988,5,-2335.86,-676.227,-9.41278,100,0,0),
+					(2988,6,-2292.79,-586.661,-9.38816,100,0,0);
+
+			-- Update Bloodhoof Village Guards with correct NPC flag so they give Guard gossip.
+				UPDATE `creature_template` SET `NpcFlags`='1' WHERE `Entry` IN (3210,3211,3212,3213,3214,3215,3217,3218,3219,3220,3221,3222,3223,3224);
+			
+			-- Update Curse of Thule target for all Rot Hide's that cast it.
+				UPDATE `creature_ai_scripts` SET `action1_param2`='1' WHERE `action1_param1`='3237' AND `action1_param2`='0';
+
+			-- UBRS Mob First Room Cleanup and Retail Adjustments for Hall of Binding
+				DELETE FROM `creature` WHERE `guid` IN (45834,160134,160136,160138,160139,160140,160141,160143,160151,160154,160155,160159,160116);
+				INSERT INTO `creature` (`guid`, `id`, `map`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `DeathState`, `MovementType`) VALUES
+					(600000, 9819, 229, 0, 0, 212.074, -335.764, 76.8696, 5.14825, 10800, 0, 0, 8807, 0, 0, 0),
+					(600001, 9819, 229, 0, 0, 125.186, -334.310, 70.9266, 4.71040, 10800, 0, 0, 8807, 0, 0, 0);
```

Comment:
One other thing I forgot, you creature table guids are auto, so you can remove `guid`   and the guid values 600000, 600001
This will prevent issues if people have costume content  that may use these values.
(PS, still got that piggy back issue upon resubmitting the PR) :-(

Reply:
I'm gonna join the discord as I'm having trouble figuring out why it's still piggybacking haha.